### PR TITLE
[WIP] ui: update ja_JP locale

### PR DESCRIPTION
**Pull request by tajisoft**
### Description

Hello dev team. I am a Japanese user. I would like to update the Japanese translation. I have one question to ask. The translated text in the json file is a Unicode string. Is it okay to decode this into a Japanese string? That makes translation conversion easier. I'm sorry I couldn't find a channel to discuss and asked a question on PR.
This is still WIP, so I'm using Draft PR.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
WIP

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
Hi @tajisoft yes I think using unicode string should be completely okay - we can as part of your PR change check the QA url for the same.

**Review by DaanHoogland — COMMENTED** [review]
looks good, but please find a japanese reviewer for the textual check

**Comment by rohityadavcloud** [comment]
cc @shwstppr can we get a Japanese translator to glance over this (maybe spend 10-15mins to quickly say LGTM proxied through you ;) )

**Comment by rohityadavcloud** [comment]
I've used the translated changes to translated via Google Translate to check, it LGTM but let's see if @shwstppr can get a LGTM as well
Screenshot 2022-04-05 at 1 23 58 PM

**Review by rohityadavcloud — APPROVED** [review]

**Comment by shwstppr** [comment]
@rohityadavcloud will do
@tajisoft will you add more changes? I see some strings are yet to be translated
Screenshot from 2022-04-05 13-44-43

**Comment by tajisoft** [comment]
@rohityadavcloud @shwstppr Thank you for comment. I'm thinking of checking all the translations. Currently it is finished until 1/4. It will take a little longer, but I will do it all. When all is done and the review is ready, change to Wait for review status.

**Comment by rohityadavcloud** [comment]
Thanks for updating @tajisoft pl ping us when your PR is ready for review.

**Comment by tajisoft** [comment]
@rohityadavcloud I got it.

**Comment by tajisoft** [comment]
@rohityadavcloud @shwstppr Translation is complete. I think this makes it easier for Japanese users. Unfortunately I couldn't find another Japanese developer so I couldn't double check. I will continue to fix any translations that are strange in context.

**Comment by rohityadavcloud** [comment]
Thanks @tajisoft I'll see if we've somebody to review your changes, @shwstppr can we request your translator friend to spare 5-10mins to skim/check this PR 😉?

**Review by DaanHoogland — APPROVED** [review]

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:30]:
```suggestion
"firewall.close": "ファイアウォール閉め",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:48]:
```suggestion
"label.access": "アクセス",
```

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json]: @alkasingh03 Thank you for review!!
"reject" -> "拒否"
"accept" -> "承諾"
I think this pair would be good.

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:461]: デフォルトとして

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:466]:
```suggestion
"label.assigning.vms": "VMを割り当て中",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:550]:
```suggestion
"label.certchain": "チェイン",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:947]:
```suggestion
"label.example.plugin": "例プラッグイン",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:962]:
```suggestion
"label.files": "取得する代替ファイル",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:975]:
```suggestion
"label.forbidden": "禁断",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:980]:
```suggestion
"label.free": "自由",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1140]:
```suggestion
"label.instancename": "インスタンス名",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1350]:
```suggestion
"label.make": "作成",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1618]:
```suggestion
"label.overprovisionfactor": "オーバープロビジョニング要因",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1814]:
```suggestion
"label.read": "読み取り",
```

File: ui/public/locales/ja_JP.json
Comment (DaanHoogland) [reply ui/public/locales/ja_JP.json:461]: @alkasingh03 , do you mean that 
```suggestion
"label.as.default": "デフォルトとして",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1815]:
```suggestion
"label.read.io": "読み取り (IO)",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1816]:
```suggestion
"label.readonly": "読み取り専用",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:2049]:
```suggestion
"label.sharewith": "共有",
```

File: ui/public/locales/ja_JP.json
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:2051]:
```suggestion
"label.showing": "表示中",
```

**Review by alkasingh03 — COMMENTED** [review]
＠tajisoft some more suggestions added.
Good work translating huge list! よく頑張りましたね。

**Comment by alkasingh03** [comment]
@rohityadavcloud Rohit-san reviewed changes, added suggestions for some missing strings :)

**Comment by tajisoft** [comment]
> ＠tajisoft some more suggestions added. Good work translating huge list! よく頑張りましたね。

Thank you for review and a lot of suggestions!! I'll fix it soon!

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:975]: The word "forbidden" is so well known that I decided not to translate it.

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:1140]: @alkasingh03 
English is using "Internal name" here. I think [Internal = "内部" ] is good. What do you think?
「"label.instancename": "Internal name",」

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:446]: I use "アノテーター" here.

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:947]: I use "プラグイン例" here.

File: ui/public/locales/ja_JP.json
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:1140]: I keep to use "内部名" here.

**Comment by tajisoft** [comment]
Thank you to everyone who cooperated! I'm very happy that the translation has been completed and it has become more friendly to Japanese users!

**Comment by tajisoft** [comment]
I think I made a mistake in git operation. Can I open a new PR?

**Comment by tajisoft** [comment]
I opened a new PR #6332.

**Comment by DaanHoogland** [comment]
> I opened a new PR #6332.

thanks @tajisoft , most of the time git mistakes are repairable. Iĺl close this PR as you already opened a new one.

**Comment by tajisoft** [comment]
> > I opened a new PR #6332.
> 
> thanks @tajisoft , most of the time git mistakes are repairable. Iĺl close this PR as you already opened a new one.

@DaanHoogland  Thank you for comment! Next up will improve my Git skills!

**Pull request closed without merging**
